integration-tests/jest/jest-utils.js

```diff
@@ -89,21 +88,21 @@ const flakeShaker = ({ fn, name }) => {
     `Timeout (minutes): ${clusterTimeout} (if your test times out adjust CLUSTER_TIMEOUT_MINUTES in your environment)`,
     '']
 
-  rawConsole.log(`\n\x1b[1m${/* bold */ '╔'.padEnd(padding, '═')}╗`)
-  rawConsole.log(`${`║ Running flake shaker on ${name} to flush out test flakiness...`.padEnd(padding)}║`)
-  rawConsole.log(`${'╚'.padEnd(padding, '═')}╝`)
+  console.log(`\n\x1b[1m${/* bold */ '╔'.padEnd(padding, '═')}╗`)
```

Comment:
Perhaps we should keep these as rawConsole because the flaky test runner is only run by folks locally. I believe that switching these to console will make the output when running locally quite noisy.

Reply:
Yes, please do keep the "test-flakes" messages using rawConsole (I tried it out locally).

Reply:
Good point 👍🏼

Reply:
You're right. It makes sense to keep rawConsole in this function.